### PR TITLE
[ExtensionSamples] Make the Debug configuration not use distribution certificates.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: ExtensionSamples/ExtensionSamples/ExtensionSamples.csproj [hunk 1]
```diff
@@ -31,7 +31,6 @@
     <Profiling>true</Profiling>
     <PlatformTarget>x86</PlatformTarget>
     <CodeSignProvision>Automatic</CodeSignProvision>
-    <PackageSigningKey>3rd Party Mac Developer Installer</PackageSigningKey>
```
Comment (mykyta-bondarenko-gl) [inline ExtensionSamples/ExtensionSamples/ExtensionSamples.csproj]: It's a default value that VSfM adds after you opened the project signing window. Should we remove it?

Comment (rolfbjarne) [reply ExtensionSamples/ExtensionSamples/ExtensionSamples.csproj]: @mykyta-bondarenko-gl VSfM adds it even for Debug configurations?

Comment (mykyta-bondarenko-gl) [reply ExtensionSamples/ExtensionSamples/ExtensionSamples.csproj]: @rolfbjarne yes, it does. Checked on this PR.

Comment (rolfbjarne) [reply ExtensionSamples/ExtensionSamples/ExtensionSamples.csproj]: @mykyta-bondarenko-gl OK, I've reverted this particular change.

That said, VSfM shouldn't write out default values 😒

**Pull request merged**